pkgs/by-name/mi/mingtest/package.nix

```diff
@@ -0,0 +1,25 @@
+{
+  stdenv,
+  cmake,
+  texliveFull
+  }:
+
+  stdenv.mkDerivation {
+     name = "mingtest";
+     version = "0.1.9-1";
+     src = fetchGit {
+        url = "https://github.com/craflin/mingtest.git";
+        ref = "master";
+  };
+
+  postPatch = ''
+     sed -i -e 's$include(CDeploy)$$' CMakeLists.txt
+     sed -i -e 's$install_deploy_export()$$' CMakeLists.txt
+  '';
+
+  buildPhase =''
+  '';
+
+  buildInputs = [ cmake texliveFull ];
```

Comment:
Are you serious about that?
Did you know texliveFull has a 4+G size?

Reply:
Ok, I tryed to load limited packages for texlive, but still stylefiles was missing. I will try to strip it down. In the fist run, it was the fastest to get it running with texlifvFull.

Reply:
using texLIveSmall is indeed sufficieant.